cmd/node/node.go

```diff
@@ -297,6 +300,15 @@ func (app *App) Initialize() (err error) {
 	if err := os.MkdirAll(app.Config.DataDir(), 0o700); err != nil {
 		return fmt.Errorf("ensure folders exist: %w", err)
 	}
+	lockName := filepath.Join(app.Config.DataDir(), lockFile)
+	fl := flock.New(lockName)
+	locked, err := fl.TryLock()
```

Comment:
curious about the difference btwn using flock package and using os package directly to create an empty file

Reply:
@countvonzero it's not only about creation of empty file. It's atomic file lock. (on some platforms it will create directory instead because of API differences).

That should prohibit running two instances even in racy-ish conditions.

Reply:
oh. i guess i am only thinking about linux OS. thanks